ui/hooks/useRetryTransaction.js

```diff
@@ -29,6 +26,10 @@ import { useIncrementedGasFees } from './useIncrementedGasFees';
 export function useRetryTransaction(transactionGroup) {
   const { primaryTransaction } = transactionGroup;
   const isMainnet = useSelector(getIsMainnet);
+
+  // Starts polling for gas prices that will be used for retry gas pricing
+  useGasFeeEstimates();
```

Comment:
This might be a bit nitpicky: it feels like it would be nice to separate the "get gas fee estimates" concern from the "start polling" concern. Calling `useGasFeeEstimates()` in some cases for the returned data, and in other cases for the side effect of starting polling, seems like it could be a bit confusing to future readers/maintainers

Reply:
But also, why is `useGasFeeEstimates` called here and not in a component?

Reply:
And is this also needed for cancels?

Reply:
agreed, but the good news is this is a temporary measure until users are using the new UI.

Reply:
I can just copy/paste the polling code from that hook here instead

Reply:
> But also, why is useGasFeeEstimates called here and not in a component?

I'm having a hard time remembering why I did this, though it is fine to call a hook from within a hook. Thats part of their power in composability. I do think here it'd be more appropriate to just poll -- however, like I said I'm having hard time remembering why this was necessary.

I did remove the setCustomGasPriceForRetry method that calls the fetch code, but I think this is covered by having a polling creator in the gas modal so this likely can just be removed.

Reply:
removed in latest